src/librustc_back/target/json/aarch64-apple-ios.json

```diff
@@ -0,0 +1,28 @@
+{
+  "arch": "aarch64",
+  "archive-format": "bsd",
+  "cpu": "cyclone",
+  "data-layout": "e-m:o-i64:64-i128:128-n32:64-S128",
+  "dll-suffix": ".dylib",
+  "eliminate-frame-pointer": false,
+  "env": "",
+  "exe-allocation-crate": "alloc_jemalloc",
+  "executables": true,
+  "features": "+neon,+fp-armv8,+cyclone",
+  "function-sections": false,
+  "has-rpath": true,
+  "is-builtin": true,
+  "is-like-osx": true,
+  "llvm-target": "arm64-apple-ios",
+  "max-atomic-width": 128,
+  "os": "ios",
+  "pre-link-args": [
+    "-arch",
+    "arm64",
+    "-Wl,-syslibroot",
+    "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS9.3.sdk"
```

Comment:
Hm I don't think this is the kind of path that we can encode here. The compiler should still discover this dynamically, although it can perhaps do so outside the scope of custom target specifications.